### ISSUE TITLE
[BUG] Attachments not saving

### ISSUE THREAD

**Issue by Homer625** [taigaio/taiga-front#108]
I need to upgrade Taiga from our old version and move it into docker. I have setup the Taiga site in docker, using the instructions here: https://docs.taiga.io/setup-production.html

After some tweaks to make it work in my environment, the site is up and running. I was able restore the database, media, and static folders from the old server to the new with success. Everything works great except I cannot add attachments to issues. I am able to add the attachment (see the Linux machines.xlsx in the screenshot below) but when I click on it there's no file there:

![Attachments](https://user-images.githubusercontent.com/85648401/137986486-c14e191f-ced9-4a36-9c3d-3f72642ff616.png)

I mouse over the link and it appears to have saved but when I check the file system, and the container file system, the directory and file do not exist. In my example below, there should be an "f" directory under "1":

![Attachments2](https://user-images.githubusercontent.com/85648401/137985436-57eaf524-2e27-43a9-bfd6-c94b0df418ce.png)

I confirmed that the host machine has the correct permissions on the volume folder:

![Attachments3](https://user-images.githubusercontent.com/85648401/137986049-ecd47a82-d26d-4137-8e5c-6efb1d7e686b.png)

taiga-back and taiga-gateway have the same volume syntax:

![Attachments4](https://user-images.githubusercontent.com/85648401/137986236-85760771-1bb5-4ee6-858e-c568c8ad3d8f.png)

I am running this with the latest stable versions of all the docker components required on Ubuntu 20.04.

Any ideas? I only found one other instance of this happening but it had no solution. Thanks!

**Comment by bameda** [comment]
Can you paste logs from taiga-back? I don't understand how an url of a file that does not exist can be formed, nor how the creation call has not failed if the file has not been created.

**Comment by Homer625** [comment]
Thanks for your response. I uploaded another file attachment and the taiga-back log was not updated with anything. The last entry in the log file is from a user doing a password reset a few days ago. Is there another log I can check?

**Comment by bameda** [comment]
No, running the doker-compose without -d should be enough to see all the logs

**Comment by Homer625** [comment]
The command I ran was: docker logs taiga-docker_taiga-back_1

That gave me results but did not show any sort of log of files being uploaded

**Comment by bameda** [comment]
Maybe in the taiga_async container??? or in the rabbit container??

**Comment by Homer625** [comment]
I looked at the taiga_async container log and all it showed was the email notification html template that got sent to the ticker owner.

Here is a screenshot of the rabbit container log from that time period:
![rabbitlog](https://user-images.githubusercontent.com/85648401/141141591-28472545-fe60-4d7a-9309-aaa1e989337a.png)
